_data/sites.json

```
@@ -9128,6 +9128,16 @@
         ]
     },
 
+    {
+        "name": "Webex - Free Account",
+        "url": "https://settings.webex.com",
+        "difficulty": "easy",
+        "notes": "Instructions at https://help.webex.com/en-us/5m4i4y/Delete-Your-Free-Webex-Account#id_111643",
```

```suggestion
        "url": "https://help.webex.com/en-us/5m4i4y/Delete-Your-Free-Webex-Account#id_111643",
        "difficulty": "easy",
        "notes": "Just follow the settings to find the delete account button and click it",
```

The helper link has the actual link to be used + instructions, so it better be the link that becomes clickable for anyone not reading the notes.